GettingStartedDeb.md

```diff
@@ -125,3 +125,9 @@ The NuGet.config file should look something like the following:
 </configuration>
 ```
 The important part of this is that you have a package source with aspnetvnext and nuget.org in it.
+
+Nuget restore also checks the hash on the package. So ensure that you have `libcrypto` available. To do this run: 

```

Comment:
If you guys have another fix in the pipe then I can close this. But to me it seems like you need libssl-dev to get this working.